scss/style.scss

```diff
@@ -1,23 +1,138 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+    background-color: #f0ecec;
+    padding-top: 50px;
+}
+
+.calculator {
+    
+    width: 330px;
+    height: 430px;
+    background-color: #797979;
```

Comment:
I see that you use `#797979` color three times. I would suggest extracting it to separate variable (e.g. `$darkGrey`) and use it where needed. See SASS docs: https://sass-lang.com/documentation/variables.